save-cloud-charts/save-cloud/templates/s3-deployment.yaml

```diff
@@ -0,0 +1,44 @@
+{{- if not .Values.s3.external -}}
+
+  {{- if eq .Values.profile "dev" -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: s3-secrets
+stringData:
+  s3-storage.endpoint: "http://s3:9000"
+  s3-storage.bucketName: "cnb"
+  s3-storage.credentials.accessKeyId: "admin"
+  s3-storage.credentials.secretAccessKey: "12345678"
+
+---
+  {{ end }}
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: s3
+  annotations:
+    io.kompose.service: s3
+spec:
+  selector:
+    matchLabels:
+      io.kompose.service: s3
+  template:
+    metadata:
+      labels:
+        io.kompose.service: s3
+    spec:
+      containers:
+        - image: minio/minio:latest
+          name: minio
+          ports:
+            - containerPort: 9000
+              name: minio
+          env:
+            - name: MINIO_ROOT_USER
+              value: "admin"
```

Comment:
suggestion to extract it to values and reuse in secret and here. actually I think we can use secret to set env variable for deploment

Reply:
Let's try

Reply:
Extracted such data to `values-minikube.yaml`